descriptors/postgres/debezium-postgres-1.9.0.Alpha1.json

```diff
@@ -157,57 +80,6 @@
           "x-name" : "slot.name",
           "x-category" : "CONNECTION_ADVANCED_REPLICATION"
         },
-        "slot.stream.params" : {
```

Comment:
@gunnarmorling I am not sure how often `slot.stream.params` are used. include or exclude?